var/spack/repos/spack_repo/builtin/packages/qgis/package.py

```diff
@@ -17,19 +17,22 @@ class Qgis(CMakePackage):
     maintainers("adamjstewart", "Sinan81")
 
     license("GPL-2.0-or-later")
+
+    # Prefer latest LTR
+    version(
+        "3.40.6",
+        sha256="dd68d39a2a29326031195bed2125e8b0fa7112fe9ee74d5f9850d06b02cef6a8",
+        preferred=True,
```

Comment:
`preferred=True` is technically not needed since we always default to the latest non-deprecated stable version number.